generator/simplecpp/simplecpp.h

```diff
@@ -296,6 +349,8 @@ namespace simplecpp {
         }
 
     private:
+        TokenList(const unsigned char* data, std::size_t size, std::vector<std::string> &filenames, const std::string &filename, OutputList *outputList, int unused);
```

Comment:
I'm probably missing something here - why the `unused` member instead of just changing the name of the private function?

Reply:
I suspect this was motivated to help hide the "unsafe" API:
* https://github.com/danmar/simplecpp/pull/496

Reply:
Thanks! I don't actually see the need, but it doesn't really matter.